babel/messages/extract.py

```diff
@@ -139,7 +139,7 @@ def extract_from_dir(dirname=None, method_map=DEFAULT_MAPPING,
     for root, dirnames, filenames in os.walk(absname):
         dirnames[:] = [
             subdir for subdir in dirnames
-            if not (subdir.startswith('.') or subdir.startswith('_'))
+            if not (subdir.startswith('.') or subdir == '_')
```

Comment:
Your PR does not make sense.

```
>>> a = "_"
>>> a.startswith("_")
True
>>>
```

Reply:
Please, consider reading the issue tread attached ([this one](https://github.com/python-babel/babel/issues/53)).
In my PR description, I didn't clarify what exactly I tried to fix - that's my fault.